opm/parser/eclipse/EclipseState/Schedule/OilVaporizationProperties.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef DRSDT_HPP
+#define DRSDT_HPP
+#include <iostream>
+#include <string>
+#include <memory>
+#include <opm/parser/eclipse/EclipseState/Schedule/ScheduleEnums.hpp>
+
+namespace Opm
+{
```

Comment:
This class implements a "tristate" between three different keywords; brief documentation of this slightly funny behavior would be good.